lib/build_runner.zig

```diff
@@ -948,8 +972,10 @@ fn usage(builder: *std.Build, already_ran_build: bool, out_stream: anytype) !voi
         \\  -l, --list-steps             Print available steps
         \\  --verbose                    Print commands before executing them
         \\  --color [auto|off|on]        Enable or disable colored error messages
-        \\  -fsummary                    Print the build summary, even on success
-        \\  -fno-summary                 Omit the build summary, even on failure
+        \\  --summary=[mode]             Control the printing of the build summary
```

Comment:
this looks like it does not match the actual CLI parsing code which expects the mode as the next argument.

Reply:
I planned on doing it like that but followed `--color` instead.